[2025-01-28T06:03:39.541+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-28T06:03:39.560+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_dag.check_api_ready scheduled__2025-01-27T00:00:00+00:00 [queued]>
[2025-01-28T06:03:39.565+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_dag.check_api_ready scheduled__2025-01-27T00:00:00+00:00 [queued]>
[2025-01-28T06:03:39.566+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 3
[2025-01-28T06:03:39.586+0000] {taskinstance.py:2889} INFO - Executing <Task(HttpSensor): check_api_ready> on 2025-01-27 00:00:00+00:00
[2025-01-28T06:03:39.591+0000] {standard_task_runner.py:72} INFO - Started process 2477 to run task
[2025-01-28T06:03:39.594+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'weather_dag', 'check_api_ready', 'scheduled__2025-01-27T00:00:00+00:00', '--job-id', '9', '--raw', '--subdir', '/home/ubuntu/airflow/dags/weather_dag.py', '--cfg-path', '/tmp/tmppif0d9cb']
[2025-01-28T06:03:39.596+0000] {standard_task_runner.py:105} INFO - Job 9: Subtask check_api_ready
[2025-01-28T06:03:39.634+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_dag.check_api_ready scheduled__2025-01-27T00:00:00+00:00 [running]> on host ip-172-31-38-205.ap-southeast-2.compute.internal
[2025-01-28T06:03:39.693+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='trinhminhquan2810@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='weather_dag' AIRFLOW_CTX_TASK_ID='check_api_ready' AIRFLOW_CTX_EXECUTION_DATE='2025-01-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-27T00:00:00+00:00'
[2025-01-28T06:03:39.694+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-28T06:03:39.694+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-28T06:03:39.695+0000] {logging_mixin.py:190} INFO - Current task name:check_api_ready state:running start_date:2025-01-28 06:03:39.561494+00:00
[2025-01-28T06:03:39.695+0000] {logging_mixin.py:190} INFO - Dag name:weather_dag and current dag run status:running
[2025-01-28T06:03:39.695+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-28T06:03:39.696+0000] {baseoperator.py:421} WARNING - HttpSensor.execute cannot be called outside TaskInstance!
[2025-01-28T06:03:39.696+0000] {http.py:140} INFO - Poking: /data/2.5/weather?q=Perth&APPID=dd89218a997dc67bbb95545d99c64232
[2025-01-28T06:03:39.849+0000] {base.py:84} INFO - Retrieving connection 'weathermap_api'
[2025-01-28T06:03:40.046+0000] {base.py:336} INFO - Success criteria met. Exiting.
[2025-01-28T06:03:40.050+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-28T06:03:40.051+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_dag, task_id=check_api_ready, run_id=scheduled__2025-01-27T00:00:00+00:00, execution_date=20250127T000000, start_date=20250128T060339, end_date=20250128T060340
[2025-01-28T06:03:40.067+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-28T06:03:40.067+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-28T06:03:40.067+0000] {logging_mixin.py:190} INFO - Dag name:weather_dag queued_at:2025-01-28 06:03:35.904746+00:00
[2025-01-28T06:03:40.067+0000] {logging_mixin.py:190} INFO - Task hostname:ip-172-31-38-205.ap-southeast-2.compute.internal operator:HttpSensor
[2025-01-28T06:03:40.096+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-28T06:03:40.118+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-28T06:03:40.126+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
